lang/spec.html

```diff
@@ -5932,8 +5932,8 @@ <h3>Additive expression</h3>
 </li>
 </ul>
 <p>
-If the contextually expected type for a multiplicative-expr is T, then the
-contextualy expected type for both operand expressions is T', where a value v is
+If the contextually expected type for a additive-expr is T, then the
```

Comment:
I'm not sure if this should be
```suggestion
If the contextually expected type for an additive-expr is T, then the
```